repo/packages/H/hive-metastore/0/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "0.1.0-3.0.0-beta"
+  ],
+  "downgradesTo": [
+    "0.1.0-3.0.0-beta"
+  ],
+  "minDcosReleaseVersion": "1.11",
+  "name": "hive-metastore",
+  "version": "1.0.0-3.0.0",
+  "maintainer": "support@mesosphere.io",
+  "description": "Apache Hive Metastore is the central repository of Hive metadata. It stores the metadata for Hive tables and relations.",
+  "selected": true,
+  "framework": true,
```

Comment:
Can we add the `lastUpdated` field with an epoch that matches [this timestamp](https://github.com/mesosphere/universe/pull/2429/files#diff-9edd78fd17fc6b98858bbbd89312371eR38)?

Please see the convo thread here: https://github.com/mesosphere/universe/pull/2422#pullrequestreview-306261926

Basically as soon as the PR Tarun created is merged, the tooling will automatically add the field to newer packages but for this PR we have to do it manually.